app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <base-config cleartextTrafficPermitted="true">
+        <trust-anchors>
+            <certificates src="system" />
+        </trust-anchors>
+    </base-config>
+</network-security-config>
```

Comment:
@hmpace is this ok?

Reply:
No, this is a pretty major security flaw with a pretty simple solution.
We ought to be connecting through https://thinqtv.herokuapp.com, which removes the need for cleartext traffic.

Reply:
He was already connecting via https though. @anibalb2500 were you getting an error that required enabling cleartext traffic?

Reply:
Removed the entire file since it was not being implemented. The reason for it being there was because I was receiving a "net::Err_CLEARTEXT_NOT_PERMITTED" back before I understood we were connecting through https://thinqtv.herokuapp.com.